### PR TITLE
Implement Vue Framework for PPG Oyo Client

### PR DESCRIPTION
**Pull request by kegilbert**
**NOTE** CSV file submit is currently broken, but thought I'd at least get the eslint and Vue framework for the dashboard setup.

There was a lot of tearing out of stuff, easier to look at the positive changes (although worth double checking that I didn't pull out anything important).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,127 +1,65 @@
...
-      "name": "Andrzej Kopański",
-      "url": "https://github.com/xidedix"
-    }
-  ],
-  "main": "src/index.html",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/coreui/coreui-free-bootstrap-admin-template.git"
-  },
+  "name": "ppg_regus_client",
```
Comment (mrcoulter45) [inline package.json]: > "name": "ppg_regus_client",

ppg-oyo-client

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <!-- Icons-->
+    <link rel="icon" type="image/ico" href="../../public/img/favicon.ico" sizes="any" />
+    <link href="node_modules/@coreui/icons/css/coreui-icons.min.css" rel="stylesheet">
+    <link href="node_modules/flag-icon-css/css/flag-icon.min.css" rel="stylesheet">
+    <link href="node_modules/font-awesome/css/font-awesome.min.css" rel="stylesheet">
+    <link href="node_modules/simple-line-icons/css/simple-line-icons.css" rel="stylesheet">
+
+    <!-- Main styles for this application-->
+    <link href="vendors/pace-progress/css/pace.min.css" rel="stylesheet">
+
```
Comment (mrcoulter45) [inline public/index.html]: delete new line

File: src/components/Dashboard.vue [hunk 3]
```diff
@@ -618,22 +583,22 @@ <h6>System Utilization</h6>
...
+    },
+    computed: {
+    },
+    watch: {
+    },
+    mounted() {
+    },
+    methods: {
+    }
+}
```
Comment (mrcoulter45) [inline src/components/Dashboard.vue]: add new line

File: src/components/SubmitData.vue [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+                room_id: this.room_id,
+                room_type: this.room_type,
+                room_occupied: this.room_occupied,
+                member_status: this.member_status,
+                vacate: this.vacate,
+                add_time: this.add_time,
+                compliance: this.compliance,
+                submit_time: Math.round((new Date()).getTime() / 1000),
+            // eslint-disable-next-line no-unused-vars
+            }).then((data) => {});
```
Comment (mrcoulter45) [inline src/components/SubmitData.vue]: do you want this submit form stuff here?

File: src/components/SubmitData.vue [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    mounted() {
+        // start 2 min countdown timer
+        setInterval(() => {
+            if (this.countdown_timer > 0) {
+                this.countdown_timer -= 1;
+                this.countdown_display = `${parseInt(this.countdown_timer / 60, 10)}m ${this.countdown_timer % 60}s`;
+            } else {
+                this.countdown_display = 'EXPIRED';
+            }
+        }, 1000);
```
Comment (mrcoulter45) [inline src/components/SubmitData.vue]: do you want this counter stuff here?

Comment (kegilbert) [reply src/components/SubmitData.vue]: Copy pasta laziness strikes again, removing this and the other segment you commented on.

File: package.json (hunk 1 above)
Comment (mrcoulter45) [reply package.json]: actually sorry, "ppg_oyo_client" would match the others

**Comment by kegilbert** [comment]
@mrcoulter45 Updated. I had some deformed HTML in SubmitData.vue, that part seems to work now as well. 

In regards to build time, it seems to be something in `Dashboard.vue` or `SubmitData.vue`. If either of those get updated it takes minutes, not sure what's causing. I'll poke around more before merging.

**Comment by mrcoulter45** [comment]
@kegilbert  cool, and I added 1 more comment about package.json

**Comment by kegilbert** [comment]
I left it as the name of the repo, but if we're thinking of changing the repo name as well I'll just update the package file name now.

**Pull request merged**